data-products/src/main/scala/org/sunbird/analytics/exhaust/OnDemandDruidExhaustJob.scala

```diff
@@ -48,7 +51,26 @@ object OnDemandDruidExhaustJob extends optional.Application with BaseReportsJob
 
     implicit val frameworkContext: FrameworkContext = getReportingFrameworkContext()
     implicit val conf = spark.sparkContext.hadoopConfiguration
-    execute()
+    try {
+      execute()
```

Comment:
Add JobLogger.end after execute() for success scenario

Reply:
Added . Please check